src/tests/ftest/container/attribute.py

```diff
@@ -46,20 +53,31 @@ def create_data_set():
             data_set[key] = get_random_bytes(size)
         return data_set
 
-    def verify_list_attr(self, indata, size, buff, mode="sync"):
+    def verify_list_attr(self, indata, outdata, mode="sync"):
         """
-        verify the length of the Attribute names
+        Args:
+            indata: Dict used to set attr
+            outdata: Dict obtained from list attr
+            mode: sync or async
+
```

Comment:
We follow the comments as shown below:
            indata (dict): Dict used to set attr
            outdata (dict): Dict obtained from list attr
            mode (str): sync or async, Defaults to sync.

Reply:
Could you elaborate? Do you mean adding "Default to sync" within the docstrings?

Reply:
indata (dict):
outdata (dict): 
mode (str): 

Variable types should be added on the docstring and also if there is some defaults... Add them,